hr_attendance_split/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,27 @@
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.hr.attendance</field>
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="55"/>
+        <field name="inherit_id" ref="hr_attendance.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('settings')]/div[@groups='hr_attendance.group_hr_attendance_manager']/div[position()=1]" position="after">
+                <div class="row mt16 o_settings_container" name="attendance_rules" id="attendance_rules">
+                    <div class="col-xs-12 col-md-6 o_setting_box">
+                        <div class="o_setting_left_pane">
+                            <field name="split_attendance"/>
+                        </div>
+                        <div class="o_setting_right_pane">
+                            <label for="split_attendance" string="Split Overnight Attendance"/>
+                            <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." aria-label="Values set here are company-specific." groups="base.group_multi_company" role="img"/>
+                            <div class="text-muted">
+                                Choose if attendances will be split if the start/end times cross over midnight.
+                                Example: User clocks in at 11pm, then clocks out 9am next day, two attendance would be created with the first clock-out at midnight and next clock-in at 12:01am.
```

Comment:
This line is not correct is it? The module sets the check-out at 23:59, and the check-in at midnight (00:00).

Reply:
I missed this, you are correct. I corrected the wording.